spec/spec_helper_acceptance.rb

```diff
@@ -59,6 +59,16 @@
 aIZ8kxsBjLvpi9KQTHi7Wl6Sw3ecoYdKy+2P8S5xOIpWjs8XVmOWf7Tq1+9KPv3z
 HLw/FDCzntkdq3G4em15CdFlO9BTY4HXiHU=
 -----END CERTIFICATE-----" > /tmp/blah.cert'
+
+      #--- START SELINUX WORKAROUND ---
```

Comment:
What about just dropping the certs into /etc in the first place, and then just using /etc/blah.{crt,key} as the path? Then we don't have to use this whole workaround at all, assuming selinux allows those files to be in /etc
Or symlink /etc/pki to /etc/ssl early in the process for Debian?

I also see some discussion about disabling SELinux with Beaker here: https://bugzilla.redhat.com/show_bug.cgi?id=1074384